### PR TITLE
Deploy with travis

### PR DESCRIPTION
**Pull request by altafan**
This adds the deploy phase to Travis when adding a commit on vulpemventures/nigiri-chopsticks (master branch).

For PRs opened from a fork repo, the deploy phase it's not actually executed even if `scripts/deploy` is configured to test builds with docker buildx for branches different from `master` (and for pull requests).

Ideally, the Travis' deploy phase should be always executed but the build images should be published only when committing to vv master.

### CODE REVIEW

File: scripts/deploy [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+if [ "$TRAVIS_PULL_REQUEST" = "true" ] || [ "$TRAVIS_BRANCH" != "master" ]; then
+  docker buildx build \
+    --progress plain \
+    --platform=linux/amd64,linux/arm/v7 \
+    .
+  exit $?
+fi
+
+echo "$DOCKER_PASSWORD" | docker login -u "$DOCKER_USERNAME" --password-stdin &> /dev/null
```
Comment (tiero) [inline scripts/deploy:12]: How it should source the Docker Password? Travis secrets?
 We should create docker hub user just for be consumed by travis?

Comment (tiero) [reply scripts/deploy:12]: Something like this? https://docs.travis-ci.com/user/encryption-keys#usage

Comment (altafan) [reply scripts/deploy:12]: Sorry, I forgot to add the configuration part.
Anyway, it's how you said: we have to create a user in the docker hub to be consumed by travis and we have to set the environment variables from travis repo's settings.

I did the last operation from the UI, not from the CLI as you linked, and it encrypted the variables by default.

**Comment by altafan** [comment]
Some logic has been moved from the travis YAML file to `scripts/deploy`.

Now the script performs a "dry run" deployment for all branches of VV other than master, or for any fork of that repo.

In this way we actually test the deployment before actually performing it.

File: scripts/deploy (hunk 1 above)
Comment (tiero) [reply scripts/deploy:12]: Sending credentials in private

Comment (altafan) [reply scripts/deploy:12]: Did you already set the right `DOCKER_USERNAME` or I must replace it?

**Pull request merged**